pkg/interface/src/views/apps/publish/components/Note.tsx

```diff
@@ -148,4 +141,16 @@ export function Note(props: NoteProps & RouteComponentProps) {
   );
 }
 
-export default Note;
+export default function(props: NoteProps & RouteComponentProps) {
```

Comment:
Kinda curious why this fixes the hook crash

Reply:
Rules of hooks, i.e. early returns

Reply:
oh derp yep lol